### DISCUSSION TITLE
[Bug]: Nome do arquivo de instalação do drive FireDAC na IDE Delphi não encontrar o arquivo "RESTDWDriverFireDAC.bpl"

### DISCUSSION THREAD

**Discussion by delmardelima** [OpenSourceCommunityBrasil/REST-DataWare#217] [Q&A]
### Descrição do bug

Após a instalação do RestDataWare na versão 2.1, eu tive problemas ao tentar instalar o driver FireDAC na IDE Delphi. Ao tentar acessar o arquivo "RESTDWDriverFireDAC", a IDE não conseguiu encontrá-lo.
Ao verificar o projeto, percebi que o nome do arquivo era diferente do esperado, sendo "RESTDWFireDACDriver".
Depois de algumas tentativas, decidi renomear o arquivo no projeto para "RESTDWDriverFireDAC", e finalmente consegui instalar o driver com sucesso.
No final, consegui concluir a instalação do RestDataWare com o driver FireDAC, mas fiquei preocupado com a possibilidade de outros usuários enfrentarem o mesmo problema. Espero que a equipe de suporte possa resolver este problema para evitar que outros usuários enfrentem os mesmos desafios.

### Passos para reproduzir o problema

Instalação do Drive do FireDAC

### Comportamento esperado

A IDE Delphi não encontra o arquivo "RESTDWDriverFireDAC"

### Versão RDW:

2.1.0 (branch)

### IDE:

Delphi novo

### Sistema Operacional:

Windows

### Relator do Problema

@delmardelima

**Comment by mobius1qwe** [comment]
Isso não é bug, você não seguiu [os passos de atualização](https://github.com/OpenSourceCommunityBrasil/REST-DataWare/wiki/Atualiza%C3%A7%C3%A3o-2.1) pra remover totalmente a versão 2.0 anterior antes de instalar a versão 2.1 que tem nomes diferentes nos drivers.

**Reply by delmardelima** [reply]
Assim, entendi!
desculpa, estava ausente do projeto e não percebir as novas atualização. vou desistalar tudo e refazer seguindo Wiki.
Muito Obrigado!
